opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -49,8 +49,8 @@
 Environment variables used by OpenTelemetry
 -------------------------------------------
 
-1. OPENTELEMETRY_PYTHON_METER_PROVIDER
-2. OPENTELEMETRY_PYTHON_TRACER_PROVIDER
+1. OTEL_METER_PROVIDER
```

Comment:
should this be "OTEL_PYTHON_METER_PROVIDER"?

Reply:
Fixed